kafka-ui-react-app/src/lib/hooks/useConfirm.ts

```diff
@@ -1,12 +1,13 @@
 import { ConfirmContext } from 'components/contexts/ConfirmContext';
 import React, { useContext } from 'react';
 
-export const useConfirm = () => {
+export const useConfirm = (danger = false) => {
```

Comment:
what if we use it , in two different component that gets rendered at the same time,
one needs to be `danger` and the other does not , is the `Context` is going to provide you with that flexibility for this particular case ?  how do we handle it.

Reply:
Because instance for each useConfirm is a different link behavior for each will be different, according to `danger` param

Reply:
correct me if i am wrong

but aren't they sharing the same `context`, or we are thinking of separating the context for each one, but in that case it will take a lot of boilerplate code to use this.